BaseFilter/sections/specific.txt

```diff
@@ -280,6 +280,8 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+||atube.xxx/static/nblock/
+atube.xxx##a[onclick^="changeLocation('/static/go.html?"]
```

Comment:
I still see the leftovers, so I would exchange the last rule to `atube.xxx##.sp-i`.